plots/mock/README.md

```diff
@@ -0,0 +1,25 @@
+# Mock plots
+
+-Is there a relationship between predator size (mass and length) and prey size (mass and length)?
+
+Caption: 
+Line graph representing a positive correlation between predator size (mass /length) and prey size (mass / length)
```

Comment:
Might just be personal preference but I don't think we need to include the type of graph it is in the caption. It can just be something like : "The relationship between predator size (mass/length) and prey size (mass length)."